apps/server/src/modules/account/account-api.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common';
+import { PermissionService } from '@shared/domain';
+import { UserRepo } from '@shared/repo';
+import { AccountModule } from './account.module';
+import { AccountController } from './controller/account.controller';
+import { AccountUc } from './uc/account.uc';
+import { LoggerModule } from '../../core/logger/logger.module';
+
+@Module({
+	imports: [AccountModule, LoggerModule],
+	providers: [UserRepo, PermissionService, AccountUc],
```

Comment:
Could we use the UserModule and UserService 'getUser' instead to avoid having the user repo provider?